packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message-modal.scss

```diff
@@ -0,0 +1,167 @@
+@mixin link-container-gaps {
+    padding: 1rem 0.5rem;
+    margin-right: 0.5rem;
+}
+
+#warning_scam_message_modal {
+    overflow: auto;
+}
+
+.warning-scam-message {
+    padding: 2.4rem 0;
+
+    @include mobile {
+        height: 80vh;
+        overflow: auto;
+    }
+
+    &--content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        &__title {
+            display: flex;
+            flex-direction: column;
+            margin: 2.6rem 0 2.4rem;
+
+            @media screen and (max-width: 768px) {
+                width: 80%;
+            }
+
+            @media screen and (max-width: 430px) {
+                width: 90%;
+            }
+        }
+
+        &__info {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            border: solid $color-grey-4;
+            border-radius: 0.5rem;
+            margin-top: 0.625rem;
+            padding: 1.6rem;
+            background-color: $color-grey-2;
+            width: 90%;
+
+            &--dark-mode {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+                border-radius: 0.5rem;
+                margin-top: 0.625rem;
+                padding: 1.6rem;
+                width: 90%;
+                border: solid $color-black;
+                background-color: var(--general-main-2);
+            }
```

Comment:
Why dark theme has different setup? 
by right if you use Theme.scss correctly, you don't need to define anything for dark theme